onnx/defs/nn/defs.cc

```diff
@@ -2779,64 +2776,77 @@ ONNX_OPERATOR_SET_SCHEMA(
             "T",
             {"tensor(float16)", "tensor(float)", "tensor(double)", "tensor(bfloat16)"},
```

Comment:
I suggest using `OpSchema::all_float_types_ir4` or `OpSchema::all_float_types_ir9` for consistency